packages/client/tests/functional/_example/tests.ts

```diff
@@ -1,22 +1,23 @@
-import { getTestSuiteSchema } from '../_utils/getTestSuiteInfo'
+// import { getTestSuiteSchema } from '../_utils/getTestSuiteInfo'
 import { setupTestSuiteMatrix } from '../_utils/setupTestSuiteMatrix'
 
 // @ts-ignore this is just for type checks
 declare let prisma: import('@prisma/client').PrismaClient
 
-setupTestSuiteMatrix((suiteConfig, suiteMeta) => {
+// setupTestSuiteMatrix((suiteConfig, suiteMeta) => {
+setupTestSuiteMatrix(() => {
   // an example of how to query with the preloaded client
   test('findMany', async () => {
     await prisma.user.findMany()
   })
 
-  // take a look at the test suite config (see _matrix.ts)
-  test('suiteConfig', () => {
-    console.log(suiteConfig)
-  })
+  // // take a look at the test suite config (see _matrix.ts)
+  // test('suiteConfig', () => {
```

Comment:
I wonder if we should just delete this code instead of commenting it out - back when it was the only example it sort of made sense, but right now we have a several actual tests with example `suiteConfig` usages and `suiteMeta` is fully typed.

Reply:
I think it serves a good purpose because it provides a really basic template with a simple example of where the parameters come from plus what they look like.

Reply:
I'd probably prefer to leave it uncommented and replace `console.log`s with some useful assertions about `suiteConfig` and `getTestSuiteSchema`. They would serve both as a form of documentation and as some simple checks that the testing code works properly.

Reply:
Added some simple assertions.